amida-tech/greyscale
# task/INBA-582-PO-Adds check for certain conditions before deleting subject

Makes sure that certain conditions are met before a Subject can be deleted- See all conditions here:
https://paper.dropbox.com/doc/Indaba-Conditional-Limits-ublG6ZqtvItRL1LL924Zd#:uid=093953831492899346329275&h2=Conditional-Delete

Key things to test are:
-  Deleting Subject from Project, make sure it is only deleted from the subject
- Delete from the "All Subject's" tab, make sure it is deleted from everywhere
-  Try this with an active project
- Try this with a project that has tasks assigned.
etc etc.
